packages/matchbox/src/components/LabelValue/LabelValue.js

```diff
@@ -12,20 +12,27 @@ const StyledWrapper = styled.div`
   ${margin}
 `;
 
-const Label = ({ children, orientation }) => (
-  <Box fontSize="200" fontWeight="semibold" mb={orientation === 'vertical' ? '100' : ''}>
+const Label = ({ children, orientation, appearance }) => (
+  <Box
+    fontSize="200"
+    fontWeight="semibold"
+    mb={orientation === 'vertical' ? '100' : ''}
+    color={appearance == 'inverted' ? 'white' : ''}
```

Comment:
I just realized that the label is not actually white. It should be one of the grays
<img width="229" alt="Screen Shot 2021-02-24 at 5 21 53 PM" src="https://user-images.githubusercontent.com/3903325/109073758-d7231400-76c4-11eb-95a0-ef8dd61edb1d.png">